### ISSUE TITLE
[Accessibility] [Tab] Scrollspy tabs > When card 2 content is read card 2 tab is not auto selected.

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#1949]
[A11Y-626](https://jira.cengage.com/browse/A11Y-626)

**Environment:**
Windows 11/Chrome 137/JAWS 2025
Windows 11/Edge 136/NVDA 2025

**Steps to Reproduce:**
1. Open the URL – https://react-magma.cengage.com/version/4.9.1/api/tabs/
2. Turn on the screen reader (JAWS/NVDA).
3. Navigate to the 'Tab' component page from the left navigation.
4. Navigate to "Scrollspy tabs" element and using arrow keys go to card 2 content and verify that card 2 is not auto selected

**Actual Result:**
When card 2 content is read the card 2 tab is not auto selected

Refer to screencast:
https://app.screencast.com/fAxk23MNv0O9M

**Expected Result:**
As the card 2 content is read the tab card 2 should be selected.

**User Impact:**
as the card 2 is not selected screen reader user can misunderstand the content.

WCAG Guideline: 1.3.1 info and relationship

**Bug Fix recommendation:**
Use JS function to auto select when the card 2 content is read by screen reader

**Comment by nikitaorliak-cengage** [comment]
The navigation on the left is designed to move through the content, so auto-selecting sections based on the screen reader can cause problems. May require further consideration.
